sdk/helper/random/registry.go

```diff
@@ -0,0 +1,34 @@
+package random
+
+import (
+	"fmt"
+)
+
+type ruleConstructor func(map[string]interface{}) (Rule, error)
+
+var (
+	// defaultRuleNameMapping is the default mapping of HCL rule names to the appropriate rule constructor.
+	// Add to this map when adding a new Rule type to be recognized in HCL.
+	defaultRuleNameMapping = map[string]ruleConstructor{
+		"Charset": ParseCharset,
```

Comment:
Would it be possible to lowercase `"Charset"`? I ask because I've only seen keys lowercased in the Vault config and in policies.

Reply:
:+1: